dp/jobqueueing/gcreset.py

```diff
@@ -0,0 +1,96 @@
+"""
+Script to update the status of a queue entry.
```

Comment:
Can you be more specific (either here or in the `ArgumentParser` help message) about what it's updating the status to be? Or what information it's using to "reset" the status of a queue entry? In `main()` you mention using an e-mail message as input, but what's the intended workflow. Save e-mail to disk and then point this script to the output file? If so, it's not overwhelmingly clear without reading the code in-depth.